test/camelcase.js

```diff
@@ -0,0 +1,18 @@
+var assert = require('assert');
+var requireDir = require('..');
+
+assert.deepEqual(requireDir('./camelcase', {
+    recurse: true,
+    camelcase: true
+}), {
+    aMain: 'a',
+    'a_main': 'a',
+    subDir: {
+        aSub: 'a',
+        'a-sub': 'a'
+    },
+    'sub-dir': {
+        aSub: 'a',
+        'a-sub': 'a'
+    }
+});
```

Comment:
One note: this project isn't using Mocha or anything like that at the moment, so for consistency w/ the other test files, let's add a simple echo to the end here.

```js
console.log('Camelcase tests passed.');
```